awx/ui_next/src/screens/User/UserList/UserList.test.jsx

```diff
@@ -121,17 +121,8 @@ describe('UsersList with full permissions', () => {
   });
 
   test('Users are retrieved from the api and the components finishes loading', async () => {
-    await waitForElement(
-      wrapper,
-      'UsersList',
-      el => el.state('hasContentLoading') === true
-    );
```

Comment:
There's no check for the presence of `ContentLoading` here because it would never actually be visible with the test data we're using to mock the api (paginated data list only shows the loading component if `items` is empty, which it isn't for this test)